.travis.yml

```diff
@@ -62,28 +23,19 @@ before_install:
     - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
     - chmod +x miniconda.sh
     - ./miniconda.sh -b
-    - export PATH=/home/travis/miniconda/bin:$PATH
+    - export PATH=/home/travis/miniconda2/bin:$PATH
```

Comment:
why do you name it miniconda2?

Reply:
Current distributions of miniconda install by default to `miniconda2` instead of `miniconda`

Reply:
okay